src/main/resources/application.properties

```diff
@@ -14,11 +14,13 @@ mp.messaging.incoming.ingress.value.deserializer=org.apache.kafka.common.seriali
 quarkus.datasource.db-kind=postgresql
 quarkus.datasource.username=postgres
 quarkus.datasource.password=postgres
-quarkus.datasource.jdbc.url=jdbc:postgresql://127.0.0.1:5432/notifications
-quarkus.datasource.jdbc.max-size=4
+quarkus.datasource.jdbc=false
+quarkus.datasource.reactive.url=postgresql://127.0.0.1:5432/notifications
+quarkus.datasource.reactive.max-size=4
```

Comment:
I think we can allow for some more connections. The jdbc setting was to limit the flyway ones, but r2dbc did not have that setting.

Reply:
I fixed that on the other branch.